src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/ActionRuleEndpoint.java

```diff
@@ -78,6 +78,11 @@ public List<ActionRuleDto> findActionRulesByCollex(
       @RequestParam(value = "collectionExercise") UUID collectionExerciseId,
       @Value("#{request.getAttribute('userEmail')}") String userEmail) {
 
+      authUser.checkUserPermission(
+              userEmail,
+              collectionExercise.getSurvey().getId(),
```

Comment:
survey hasn't been defined or populated yet here. This could be changed to directly use the surveyId from the request params instead.